Math/NumberTheory/Primes/Sieve/SmoothSieve.hs

```diff
@@ -0,0 +1,80 @@
+module Math.NumberTheory.Primes.Sieve.SmoothSieve
+  ( smoothData,
+    smoothNumbers,
```

Comment:
Please put commas first, as in other modules. For example,
https://github.com/Bodigrim/arithmoi/blob/76b7f93653216f208a41691ad0a562644d340a9c/Math/NumberTheory/Primes.hs#L13-L22